build/docker/zap-baseline.py

```diff
@@ -86,6 +90,8 @@ def usage():
     print ('    -i                default rules not in the config file to INFO')
     print ('    -j                use the Ajax spider in addition to the traditional one')
     print ('    -l level          minimum level to show: PASS, IGNORE, INFO, WARN or FAIL, use with -s to hide example URLs')
+    print ('    -n context_file   context file which will be loaded prior to spidering the target')
+    print ('    -p progress_file  progress file which specifies issues that are being addressed')
```

Comment:
Ok to link to https://github.com/zaproxy/zaproxy/wiki/ZAP-Baseline-Scan and detail them there?